app/pathogen/arbovirus/dashboard/(map)/ArbovirusMap.tsx

```diff
@@ -56,12 +61,14 @@ export function ArbovirusMap(input: ArbovirusMapProps) {
         <PathogenMap
           id="arboMap"
           baseCursor=""
+          sourceId="arbo-[GENERATED-SOURCE-ID]"
```

Comment:
This is a singular `sourceId` that's used for every single layer. My concern would be that we would have to pull this out if we ever wanted to have a layer besides the one we currently have so this among a couple of other changes make it feel to me like this PR might cause issues if `layers` ever had more than one element. I think what we actually might want is to have the `sourceId`s as props on the layer in `layers` and link them that way.